### PR TITLE
Minor fix to let Sign-in Method settings show up when ldap is enabled

### PR DESCRIPTION
**Pull request by jwilander**

### CODE REVIEW

**Comment by hmhealey** [comment]
+1

**Comment by jwilander** [comment]
@hmhealey please re-review, I fixed the weird autocomplete issue by adding a hidden textbox

File: webapp/components/claim/components/email_to_ldap.jsx [hunk 1]
```diff
@@ -104,13 +104,19 @@ export default class EmailToLDAP extends React.Component {
                             }}
                         />
                     </p>
+                    <input
+                        type='text'
+                        style={{display: 'none'}}
+                        name='fakeusernameremembered'
+                    />
```
Comment (crspeller) [inline webapp/components/claim/components/email_to_ldap.jsx]: Can you explain why this is necessary?

Comment (jwilander) [inline webapp/components/claim/components/email_to_ldap.jsx]: Chrome will try to autocomplete the first password and text field it finds based on some heuristics and it was causing the LDAP ID to auto-complete the email password field. Solution is based off this [stackoverflow thread](http://stackoverflow.com/a/15917221).

**Comment by crspeller** [comment]
+1

**Pull request merged**